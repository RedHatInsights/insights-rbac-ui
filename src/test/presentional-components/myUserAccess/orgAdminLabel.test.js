import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import OrgAdminLabel from '../../../presentational-components/myUserAccess/orgAdminLabel';

describe('<OrgAdminLabel />', () => {
  it('should render correctly', () => {
    expect(toJson(mount(
        <OrgAdminLabel/>
    ))).toMatchSnapshot();
  });
});
