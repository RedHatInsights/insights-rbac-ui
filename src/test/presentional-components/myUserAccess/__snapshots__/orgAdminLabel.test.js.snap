// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrgAdminLabel /> should render correctly 1`] = `
<OrgAdminLabel>
  <Tooltip
    appendTo={[Function]}
    aria="describedby"
    boundary="window"
    className=""
    content={
      <span>
         You can manage other users' permissions with 'User access' 
      </span>
    }
    distance={15}
    enableFlip={true}
    entryDelay={500}
    exitDelay={500}
    flipBehavior={
      Array [
        "top",
        "right",
        "bottom",
        "left",
        "top",
        "right",
        "bottom",
      ]
    }
    id=""
    isAppLauncher={false}
    isContentLeftAligned={false}
    isVisible={false}
    maxWidth="18.75rem"
    position="right"
    tippyProps={Object {}}
    trigger="mouseenter focus"
    zIndex={9999}
  >
    <PopoverBase
      appendTo={[Function]}
      aria="describedby"
      arrow={true}
      boundary="window"
      content={
        <div
          className=""
          id=""
          role="tooltip"
        >
          <TooltipContent
            isLeftAligned={false}
          >
            <span>
               You can manage other users' permissions with 'User access' 
            </span>
          </TooltipContent>
        </div>
      }
      delay={
        Array [
          500,
          500,
        ]
      }
      distance={15}
      flip={true}
      flipBehavior={
        Array [
          "top",
          "right",
          "bottom",
          "left",
          "top",
          "right",
          "bottom",
        ]
      }
      isVisible={false}
      lazy={true}
      maxWidth="18.75rem"
      onCreate={[Function]}
      placement="right"
      popperOptions={
        Object {
          "modifiers": Object {
            "hide": Object {
              "enabled": true,
            },
            "preventOverflow": Object {
              "enabled": true,
            },
          },
        }
      }
      theme="pf-tooltip"
      trigger="mouseenter focus"
      zIndex={9999}
    >
      <Label
        color="blue"
      >
        <span
          className="pf-c-label pf-m-blue"
        >
          <span
            className="pf-c-label__content"
          >
             Org. Administrator 
          </span>
        </span>
      </Label>
      <Portal
        containerInfo={
          <div>
            <div
              class=""
              id=""
              role="tooltip"
            >
              <div
                class="pf-c-tooltip__content"
              >
                <span>
                   You can manage other users' permissions with 'User access' 
                </span>
              </div>
            </div>
          </div>
        }
      >
        <div
          className=""
          id=""
          role="tooltip"
        >
          <TooltipContent
            isLeftAligned={false}
          >
            <div
              className="pf-c-tooltip__content"
            >
              <span>
                 You can manage other users' permissions with 'User access' 
              </span>
            </div>
          </TooltipContent>
        </div>
      </Portal>
    </PopoverBase>
  </Tooltip>
</OrgAdminLabel>
`;
